llvm/test/MC/ELF/crel-32.s

```diff
@@ -0,0 +1,18 @@
+# REQUIRES: powerpc-registered-target
+## Test CREL for a 32-bit big-endian target.
```

Comment:
AIUI from https://example61560.wordpress.com/wp-content/uploads/2016/11/powerpc_abi.pdf Power PC uses RELA relocations. Would it be worth testing a target that uses REL relocations by default?

For example 32-bit Arm is REL.

Or have I misunderstood and when CREL is used a target can choose rel or rela.